core/hydromet_stratified.py

```diff
@@ -384,6 +384,201 @@ def Scenarios_low_and_high_S(df_runoff_SR1: pd.DataFrame, mu: float, GEV_paramet
+    if df.loc['02h', 'value']<low_2:
+        test1 = low_2
+    else:
+        test1 = df.loc['02h', 'value']
+    if df.loc['02h', 'value']> up_2:
+        test2 = up_2
+    else:
+        test2 = df.loc['02h','value']
+    if test1 > test2:
+        test3 = test1
+    else:
+        test3 = test2
+    if test2 > test3:
+        test4 = test2
+    else:
+        test4 = test3
+    if test4>up_2:
+        test_f = up_2
+    else:
+        test_f = test4
+    a3 = 2.0*(df2.at[11.5, 'ratio']-2*(0.5-0.5*test_f/df.loc['24h', 'value'])+ df2.at[10.5, 'ratio'])
+    b3 = df2.at[11.5, 'ratio']-df2.at[10.5, 'ratio']-22.0*a3
+    c3 = (0.5-0.5*test_f/df.loc['24h','value'])-121.0*a3-11.0*b3  
+    ratio_to_24h.loc[0:90, 'ratio'] = a*np.power(ratio_to_24h.loc[0:90].index/10.0, 2)+\
+                                      b*ratio_to_24h.loc[0:90].index/10.0
+    ratio_to_24h.loc[91:105, 'ratio'] = a2*np.power(ratio_to_24h.loc[91:105].index/10.0, 2)+\
+                                        b2*ratio_to_24h.loc[91:105].index/10.0
+    ratio_to_24h.loc[106:115, 'ratio'] = a3*np.power(ratio_to_24h.loc[106:115].index/10.0, 2)+\
+                                         b3*ratio_to_24h.loc[106:115].index/10.0 + c3
+    ratio_to_24h['slope'] = ratio_to_24h['ratio'].diff()/0.1                                                               
+    if -0.867*ratio_to_24h.loc[115, 'slope']+0.4337 < 0.399: 
+        fac_116 = -0.867*ratio_to_24h.loc[115, 'slope']+0.4337
+    else:
+        fac_116 = 0.399
+    if -0.4917*ratio_to_24h.loc[115,'slope']+0.8182 < 0.799: 
+        fac_117 = -0.4917*ratio_to_24h.loc[115,'slope']+0.8182
+    else:
+        fac_117 = 0.799
+    ratio_to_24h.at[116, 'ratio'] = df2.at[11.5, 'ratio']+fac_116*(df2.at[11.75,'ratio']-df2.at[11.5, 'ratio'])
+    ratio_to_24h.at[117, 'ratio'] = df2.at[11.5, 'ratio']+fac_117*(df2.at[11.75,'ratio']-df2.at[11.5, 'ratio'])                                                                                                                
+    ratio_to_24h.at[118, 'ratio'] = df2.at[11.75, 'ratio']+0.4*(df2.at[11.875,'ratio']-df2.at[11.75, 'ratio'])
+    ratio_to_24h.at[119, 'ratio'] = df2.at[11.875, 'ratio']+0.6*(df2.at[11.917,'ratio']-df2.at[11.875, 'ratio'])
+    ratio_to_24h.loc[121:240, 'ratio'] = 1-ratio_to_24h.loc[0:119, 'ratio'].sort_index(ascending=False).values
+    ratio_to_24h.loc[120, 'ratio'] = ratio_to_24h.at[121, 'ratio']-(df.at['05m', 'ratio']+1.0/5.0*
+                                                                   (df.at['10m','ratio']-df.at['05m','ratio']))
+    ratio_to_24h.loc[0, 'ratio'] = 0
+    ratio_to_24h['slope'] = ratio_to_24h['ratio'].diff()/0.1
+    ratio_to_24h.at[0, 'slope'] = 0
+    ratio_to_24h['t_step'] = ratio_to_24h.index*0.1
+    ratio_to_24h.index = ratio_to_24h.index*0.1
+    return ratio_to_24h
+
+
+def get_hyeto_input_data_nrcs(temporal_precip_table_dir: str, event: any,
```

Comment:
the event variable is no longer "any", instead it should be "int"

Reply:
Changed event variable to int. Thanks